zebra/zebra_dplane.c

```diff
@@ -1779,6 +1779,14 @@ static int kernel_dplane_process_func(struct zebra_dplane_provider *prov)
 		if (ctx == NULL)
 			break;
 
+		/* A previous provider plugin may have asked to skip the
```

Comment:
both bsd and linux set flags that are needed up in the main zebra pthread for knowing route install was successfull.  How do you plan to handle this?

Reply:
yes, if a plugin wants to do this, it needs to know what zebra expects to see when the results show up.